gc/gc-tool/TODO

```diff
@@ -0,0 +1 @@
+Add LICENSE/NOTICE for gc-tool
```

Comment:
Should we have this for every published jar?

Reply:
NB: TODO is because I know gc-tool is shading dependencies, but I didn't have time to list yet. I will do that today.

Reply:
No, I mean we should do this exercise for every publish jar, right?.. or only for "fat" jars?